### ISSUE TITLE
Before hook exception reported as skipped instead of fail

### ISSUE THREAD

**Issue by wswebcreation** [wswebcreation/multiple-cucumber-html-reporter#87]
**Environment (please complete the following information):**
 - multiple-cucumber-html-reporter: 1.11.6
 - Node.js version: 10.15.0
 - NPM version: 6.8.0
 - Platform name and version: MacOs
 - Cucucmber version: 5.1.0

**Config of multiple-cucumber-html-reporter**
Using `protractor-multiple-cucumber-html-reporter-plugin`

**Describe the bug**
When error occurs in Before hook the report shows an error on before but in summary it shows as 'skipped'

**To Reproduce**
declare Before hook that return error:

```js
Before(function (scenario, done) {
   done('error');
}
```

**Expected behavior**
Report should show the Scenario as error but shows the Scenario as Skipped

**Log**
```json
[
  {
    "keyword": "Feature",
    "name": "demo feature",
    "line": 1,
    "id": "demo-feature",
    "tags": [],
    "uri": "e2e/demo.feature",
    "elements": [
      {
        "id": "demo-feature;demo-scenario",
        "keyword": "Scenario",
        "line": 3,
        "name": "demo scenario",
        "tags": [
          {
            "name": "@debug",
            "line": 2
          }
        ],
        "type": "scenario",
        "steps": [
          {
            "keyword": "Before",
            "hidden": true,
            "match": {
              "location": "e2e/steps/common.steps.ts:31"
            },
            "result": {
              "status": "failed",
              "duration": 3000000,
              "error_message": "Error: blabla\n    at World.<anonymous> (e2e/steps/common.steps.ts:32:10)"
            }
          },
          {
            "arguments": [],
            "keyword": "Given ",
            "line": 4,
            "name": "TODO",
            "match": {
              "location": "e2e/steps/common.steps.ts:95"
            },
            "result": {
              "status": "skipped"
            }
          },
          {
            "keyword": "After",
            "hidden": true,
            "match": {
              "location": "e2e/steps/common.steps.ts:34"
            },
            "result": {
              "status": "passed",
              "duration": 426000000
            },
            "embeddings": [
            ]
          },
          {
            "keyword": "After",
            "hidden": true,
            "match": {
              "location": "node_modules/protractor-cucumber-framework/lib/resultsCapturer.js:26"
            },
            "result": {
              "status": "passed",
              "duration": 1000000
            }
          }
        ]
      }
    ]
  }
]
```

**Comment by wswebcreation** [comment]
Closing this one because it works as expected

![image](https://user-images.githubusercontent.com/11979740/58758901-b6934d80-8522-11e9-9d9f-5ce96c519b89.png)

**Comment by ofirmgr** [comment]
I don't understand why you call this is "works as expected". If the "Before" method fails then the whole Scenario should be fail as well but it is marked as skipped

**Comment by wswebcreation** [comment]
Hmm, you are right, sorry, I'll fix this

**Comment by wswebcreation** [comment]
I released version `1.11.8` https://github.com/wswebcreation/multiple-cucumber-html-reporter/releases/tag/v1.11.8 to fix this problem.

Going to close the issue now
